src/test/java/edu/ksu/canvas/attendance/controller/CourseConfigurationControllerITest.java

```diff
@@ -96,7 +103,8 @@ public void saveCourseConfiguration_HappyPath() throws Exception {
                 .param("saveCourseConfiguration", "Save Course Configuration")
                 .param("defaultMinutesPerSession", String.valueOf(expectedDefaultMinutesPerSession))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "defaultMinutesPerSession" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)